tests/test_cmd_stdin.sh

```diff
@@ -35,11 +35,8 @@ strip_header_if() {
   STRIP_HEADER="$2"
 
   if ${STRIP_HEADER}; then
-    # The following does not work on all platforms:
-    #   grep --text mdat "${FILE}"
-    #   awk -b -v RS='mdat' '{print length($0); exit}' "${FILE}"
-    # Hence the hardcoded variable below.
-    MDAT_OFFSET=1061
+    MDAT_OFFSET=$(GREP_OPTIONS="" LC_ALL=C \
```

Comment:
`LC_ALL` is key here, but since I was adding that I figured `GREP_OPTIONS` would make it more future proof. Another option would be to add `-h` explicitly to avoid any chance of the filename getting added to the output.